pyscriptjs/tests/integration/test_02_display.py

```diff
@@ -5,13 +5,14 @@
 import re
 
 import numpy as np
+import pytest
 from PIL import Image
 
-from .support import PyScriptTest, skip_worker, wait_for_render
+from .support import PyScriptTest, wait_for_render
 
 
 class TestDisplay(PyScriptTest):
-    @skip_worker("FIXME: display()")
+    @pytest.mark.skip("FIXME: display() without target is broken")
```

Comment:
that's not true ... it's working fine with current live tests

Reply:
All `display` tests without target break. Not at all only checking by id. #1658 may have fixed it but at least in yesterdays' next it wasn't working

Reply:
that change is not here, it arrived later ... all tests with `display` don't import `display` so these can't work, they all need to be re-written